### PR TITLE
add more urivariables examples

### PR DESCRIPTION
**Pull request by egekorkan**
addresses #1312

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1315.html" title="Last updated on Dec 8, 2021, 4:53 PM UTC (a36910c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1315/7343d5a...a36910c.html" title="Last updated on Dec 8, 2021, 4:53 PM UTC (a36910c)">Diff</a>

### CODE REVIEW

**Review by sebastiankb — APPROVED** [review]

**Comment by sebastiankb** [comment]
from today's TD call, decided to merge

File: index.html [hunk 1]
```diff
@@ -1377,8 +1377,8 @@ <h2>Core Vocabulary Definitions</h2>
 <li><a href="#actionaffordance"><code>ActionAffordance</code></a></li>
 <li><a href="#eventaffordance"><code>EventAffordance</code></a></li></ul></section>
 <section><h3><code>PropertyAffordance</code></h3><p>An Interaction Affordance that exposes state of the
-          Thing. This state can then be retrieved (read) and/or
-          updated (write). Things can also choose to
+          Thing. This state can then be retrieved (read) and
+          optionally updated (write). Things can also choose to
```
Comment (benfrancis) [inline index.html:1381]: @egekorkan Did you mean to revert @danielpeintner's change from #1306? I'm guessing that was an accident?

Comment (egekorkan) [reply index.html:1381]: This is indeed weird and definitely unintentional. Should I revert #1306 and reopen and remerge it?

Comment (sebastiankb) [reply index.html:1381]: I will take care of it in PR https://github.com/w3c/wot-thing-description/pull/1334

**Pull request merged**